docs/csharp/programming-guide/strings/index.md

```diff
@@ -13,7 +13,9 @@ A string is an object of type <xref:System.String> whose value is text. Internal
   
 ## string vs. System.String  
 
- In C#, the `string` keyword is an alias for <xref:System.String>. Therefore, `String` and `string` are equivalent, and you can use whichever naming convention you prefer. The `String` class provides many methods for safely creating, manipulating, and comparing strings. In addition, the C# language overloads some operators to simplify common string operations. For more information about the keyword, see [string](../../language-reference/builtin-types/reference-types.md). For more information about the type and its methods, see <xref:System.String>.  
+ In C#, the `string` keyword is an alias for <xref:System.String>. If you have `using System;` in your file, `string` and `String` are equivalent, and you can use whichever naming convention you prefer. If you don't have `using System;`, your options are `string` and `System.String`, therefore `string` is more portable and is recommended.
```

Comment:
I'd always recommend `string` regardless `using System;` is there or not. Probably refer to [.NET runtime coding style document](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md).

> We use language keywords instead of BCL types (e.g. int, string, float instead of Int32, String, Single, etc) for both type references as well as method calls (e.g. int.Parse instead of Int32.Parse). See issue [#13976](https://github.com/dotnet/runtime/issues/13976) for examples.

Reply:
What should I shorten it to? Maybe just remove the comparison altogether?

> In C#, the `string` keyword is an alias for \<xref:System.String\>. Using `string` is more portable and is recommended.

Reply:
I think @BillWagner or @IEvangelist should be able to provide guidance on what information to include here.

Reply:
Yes, we should ALWAYS ALWAYS be using the alias names. This is the guidance I'd suggest for any and all C# developers, new and old alike. The alias names are part of the language, and exist to simplify your code. Most of the time a C# file will be `using System;`, but if it's not needed -- it's nice to remove it. It actually cleans the code up. I also much rather prefer the elegant look of using the aliases as they're more distinct, and have consistent character casing with other C# specific keywords.

I was going to say exactly what @Youssef1313 did, here: https://github.com/dotnet/docs/pull/24985/files#r663539805

Reply:
```suggestion
 In C#, the `string` keyword is an alias for <xref:System.String>. Therefore, `String` and `string` are equivalent, regardless it is recommended to use the provided alias `string`. The `String` class provides many methods for safely creating, manipulating, and comparing strings. In addition, the C# language overloads some operators to simplify common string operations. For more information about the keyword, see [string](../../language-reference/builtin-types/reference-types.md). For more information about the type and its methods, see <xref:System.String>.
```